### PR TITLE
Enhancement/button refactoring

### PR DESCRIPTION
**Pull request by yleaxeman**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 > Sep 14, 2022
 
+* :tada: **Enhancement** Update button system. Getting rid of buttons with outline type classes (breaking)
```
Comment (proand) [inline CHANGELOG.md:5]: :boom: **Breaking**

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
...
 * :Nut_and_bolt: **New** Added navigation tile, CTA for main entry navigation
```
Comment (proand) [inline CHANGELOG.md:6]: :nut_and_bolt: **New**

File: src/blocks/_fhi-search.block.scss [hunk 3]
```diff
@@ -19,26 +19,24 @@
...
-
-                // TODO: remove data-whatintent when we get rid of the programmatic focus logic
-                [data-whatintent="mouse"] & {
-                    filter: invert(0);
-                }
+            }
+        }
+        &:hover {
+            text-decoration: none;
+
```
Comment (proand) [inline src/blocks/_fhi-search.block.scss:37]: Is it intentional that hover effect is removed?

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 > Sep 14, 2022
```
Comment (proand) [inline CHANGELOG.md:4]: Changes to search should also be mentioned in changelog...

**Review by proand — CHANGES_REQUESTED** [review]
Buttons looks good 👍
Just ignore hover comment if you and designer have search under control.

Changelog is the only file that need canges.

File: src/blocks/_fhi-search.block.scss (hunk 3 above)
Comment (yleaxeman) [reply src/blocks/_fhi-search.block.scss]: Hover state was not stated in Figma, but is now cleared with designer.

**Pull request merged**